langchain/callbacks/tracers/base.py

```diff
@@ -209,8 +209,9 @@ def on_tool_start(
 
         tool_run = ToolRun(
             serialized=serialized,
-            action=action.tool,
-            tool_input=action.tool_input,
+            # TODO: this is duplicate info as above, not needed.
+            action=str(serialized),
```

Comment:
I can rm this from the tracing backend and make the changes here next time we release